doc/HOWTO_DOCUMENT.rst.txt

```diff
@@ -246,13 +246,35 @@ The sections of the docstring are:
      err_msg : str or None
          Human readable error message, or None on success.
 
-6. **Other parameters**
+6. **Yields**
+
+   Explanation of the yielded values and their types. This is relevant to
+   generators only. Similar to the **Parameters** section, except the name
+   of each return value is optional. The type of each return value is
+   always required::
+
+     Yields
+     ------
+     int
+         Description of the anonymous integer return value.
+
+   If both the name and type are specified, the **Returns** section takes the
```

Comment:
Opps, can you explain a bit more?  **Other Parameters** is now item 7.  **Yields** is the new item 6. Or was there another typo?